tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -138,7 +138,7 @@ public void ConstructorsTest()
                 "https://login.microsoftonline.com/tfp/vibrob2c.onmicrosoft.com/b2c_1_b2c_signup_signin_policy/",
                 app.Authority);
             Assert.AreEqual(MsalTestConstants.ClientId, app.ClientId);
-            Assert.AreEqual("urn:ietf:wg:oauth:2.0:oob", app.RedirectUri);
+            Assert.AreEqual("urn:ietf:wg:oauth:2.0:oob", app.AppConfig.RedirectUri);
```

Comment:
>urn:ietf:wg:oauth:2.0:oob [](start = 29, length = 25)

we have a const for this in Microsoft.Identity.Client.Core.Constants -> DefaultRedirectUri